pkg/api/podsecuritypolicy/util.go

```diff
@@ -45,24 +45,9 @@ func DropDisabledFields(pspSpec, oldPSPSpec *policy.PodSecurityPolicySpec) {
 }
 
 func allowedProcMountTypesInUse(oldPSPSpec *policy.PodSecurityPolicySpec) bool {
-	if oldPSPSpec == nil {
-		return false
-	}
-
-	if oldPSPSpec.AllowedProcMountTypes != nil {
-		return true
-	}
-
-	return false
-
+	return oldPSPSpec != nil && oldPSPSpec.AllowedProcMountTypes != nil
 }
 
 func sysctlsInUse(oldPSPSpec *policy.PodSecurityPolicySpec) bool {
-	if oldPSPSpec == nil {
-		return false
-	}
-	if oldPSPSpec.AllowedUnsafeSysctls != nil || oldPSPSpec.ForbiddenSysctls != nil {
-		return true
-	}
-	return false
+	return oldPSPSpec != nil && (oldPSPSpec.AllowedUnsafeSysctls != nil || oldPSPSpec.ForbiddenSysctls != nil)
```

Comment:
I _initially_ thought that the original code is semantically more explicit and therefore easier to read than the one-liner version. 

Although,  having reviewed this change for the past 5 mins I'm now comfortable with either.

Is it possible to check that the existing code has 100% code coverage in unit tests? 
If the unit tests had 100% code coverage on the existing code and this oneliner didn't break the tests  then that would provide a good degree of comfort for accepting this change.

Reply:
I've same concerns with you about test coverage, so I ran `go test -coverprofile=coverage.out && go tool cover -html=coverage.out ` on individually. Output is here:

![Screen Shot 2020-03-11 at 21 42 30](https://user-images.githubusercontent.com/12763626/76453540-1a1ca680-63e4-11ea-92a6-bfef73363db9.png)

![Screen Shot 2020-03-11 at 21 43 02](https://user-images.githubusercontent.com/12763626/76453562-23a60e80-63e4-11ea-8120-f19e10017217.png)

![Screen Shot 2020-03-11 at 21 44 30](https://user-images.githubusercontent.com/12763626/76453577-299bef80-63e4-11ea-9291-941292c7a94d.png)

Tests are fully covers changes except pod/util.go. For pod/util.go coverage is 73.0% of statements. But in my changes on pod/util.go tests not covers only overheadInUse() and topologySpreadConstraintsInUse() functions. Other changes are covered by pod/util unit tests.

![Screen Shot 2020-03-11 at 22 08 21](https://user-images.githubusercontent.com/12763626/76454179-3b31c700-63e5-11ea-8418-b3c0c673cc09.png)

So now I'll push a commit which includes unit tests for those two methods.

![Screen Shot 2020-03-11 at 23 10 14](https://user-images.githubusercontent.com/12763626/76459906-1857e080-63ee-11ea-9a7e-97ccb58bd726.png)

Reply:
Hi Emre, very nice, thanks for this! 

If you added your new unit tests so that they provide 100% cover of the existing multi-line if statements in a separate PR first that would be welcome improvement.

Reply:
Hi Robert, thank you for your feedback. I will open a seperate pr for unit tests and will add you as reviewer.